src/main/java/core/basesyntax/MatrixOperation.java

```diff
@@ -6,18 +6,42 @@
  */
 public class MatrixOperation {
     public int[] getDiagonal(int[][] matrix) {
-        return null;
+        int[] diagonal = new int[matrix.length];
+        for (int i = 0; i < matrix.length; i++) {
+            diagonal[i] = matrix[i][i];
+        }
+        return diagonal;
     }
 
     public int[] getCounterDiagonal(int[][] matrix) {
-        return null;
+        int[] cdiagonal = new int[matrix.length];
```

Comment:
cdiagonal - what does it mean?
let's use full words for creating name of the variable